src/components/Proposal/ProposalCardContainer.tsx

```diff
@@ -192,7 +196,7 @@ class ProposalCardContainer extends React.Component<IProps, IState> {
             </div>
           </div>
         </div>
-      </div>
+      </div >
```

Comment:
? wont this break?

Reply:
I don't think so, and apparently not.  But I removed the space anyway.